domain/src/main/kotlin/no/nav/su/se/bakover/domain/behandling/Behandlingsinformasjon.kt

```diff
@@ -170,8 +151,16 @@ data class Behandlingsinformasjon(
         override fun erVilkårOppfylt(): Boolean = status == Status.VilkårOppfylt
         override fun erVilkårIkkeOppfylt(): Boolean = status == Status.VilkårIkkeOppfylt
 
-        override fun avslagsgrunn(): Avslagsgrunn? =
-            if (erVilkårIkkeOppfylt()) Avslagsgrunn.FLYKTNING else null
+        fun tilVilkår(
+            stønadsperiode: Stønadsperiode,
+            clock: Clock,
+        ): FlyktningVilkår {
+            return FlyktningVilkår.tryCreate(
+                periode = stønadsperiode.periode,
+                flyktning = this,
```

Comment:
Nå er jo endringen utført, men sånn generelt, siden Behandlingsinformasjon er tenkt slettes, ville jeg prøvd ikke å lage nye referanser til den (sende den ut som this). At (when blabla ligger her) og vi kaller tryCreate på FlykningVilkår.Innvilget eller Avslag eller Uavklart.

Reply:
Det er et veldig godt poeng! Flytta det bare inn der fordi det ble så mye "greier" i Behandlingsinformasjon. Gjør overgangene lettere dersom pilene peker vekk derfra.